journalapp/views.py

```diff
@@ -1,6 +1,36 @@
-# from pyramid.response import Response
+# coding=utf-8
+from .models import DBSession, Entry
+from .form_new import NewBlogEntryForm
+from pyramid.httpexceptions import HTTPFound, HTTPNotFound
 from pyramid.view import view_config
-from journalapp.models import query_table, query_post
+from sqlalchemy import desc
+from sqlalchemy.exc import IntegrityError
+from webob.multidict import NoVars
+
+
+def query_table():
+    return DBSession.query(Entry).order_by(desc(Entry.created))
```

Comment:
Setting this up as a class method on the Entry model might be a nice idea.  It's good to have encapsulation like this.